locale/templates/auth.ftl

```diff
@@ -0,0 +1,305 @@
+recovery-description = Click the button within the next hour to create a new password. The request came from the following device:
+recovery-action = Create new password
+
+
+# Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionPaymentExpired-subject = Credit card for { $productName } expiring soon
+subscriptionPaymentExpired-title = Your credit card is about to expire
+# Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionPaymentExpired-content = The credit card you’re using to make payments for { $productName } is about to expire.
+
+subscriptionsPaymentExpired-subject = Credit card for your subscriptions is expiring soon
+subscriptionsPaymentExpired-title = Your credit card is about to expire
+subscriptionsPaymentExpired-content = The credit card you’re using to make payments for the following subscriptions is about to expire.
+# Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionsPaymentExpired-name = { $productName }
+
+unblockCode-subject = Account authorization code
+unblockCode-title = Is this you signing in?
+unblockCode-prompt = If yes, here is the authorization code you need:
+# Variables:
+#  $unblockCode (String) - An alphanumeric code
+unblockCode-prompt-plaintext = If yes, here is the authorization code you need: { $unblockCode }
+unblockCode-report = If no, help us fend off intruders and <a data-l10n-name="reportSignInLink">report it to us</a>.
+unblockCode-report-plaintext = If no, help us fend off intruders and report it to us.
+
+verificationReminderFirst-subject = Reminder: Finish creating your account
+verificationReminderFirst-title = Welcome to the { -brand-firefox } family
+verificationReminderFirst-description = A few days ago you created a { -product-firefox-account }, but never confirmed it.
+verificationReminderFirst-sub-description = Confirm now and get technology that fights for and protects your privacy, arms you with practical knowledge, and the respect you deserve.
+confirm-email = Confirm email
+verificationReminderFirst-action = { confirm-email }
+confirm-email-plaintext = { confirm-email }:
+
+verificationReminderSecond-subject = Final reminder: Activate your account
+verificationReminderSecond-title = Still there?
+verificationReminderSecond-description = Almost a week ago you created a { -brand-firefox } Account but never verified it. We’re worried about you.
+verificationReminderSecond-sub-description = Confirm this email address to activate your account and let us know you’re okay.
+verificationReminderSecond-action = { confirm-email }
+
+verify-title = Activate the { -brand-firefox } family of products
+verify-description-common = Confirm your account and get the most out of { -brand-firefox } everywhere you sign in
+verify-description-plaintext = { verify-description-common }.
+verify-description = { verify-description-common } starting with:
```

Comment:
Never concatenate full sentences like this.

This should be two separate strings.

```
verify-description-plaintext = Confirm your account and get the most out of { -brand-firefox } everywhere you sign in.
verify-description = Confirm your account and get the most out of { -brand-firefox } everywhere you sign in starting with:
```

https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#splitting-and-composing-sentences